lib/ansible/modules/windows/win_firewall_rule.py

```diff
@@ -15,7 +15,6 @@
 version_added: "2.0"
 author:
   - Artem Zinenko (@ar7z1)
-  - Timothy Vandenbrande (@TimothyVandenbrande)
```

Comment:
This GitHub handle was active once, by removing the complete entry, you're removing information. In this case I would keep the name and the handle (maybe add a note that the Github handle is no longer active ?)